control-plane-operator/main.go

```diff
@@ -7,6 +7,7 @@ import (
 
 	"github.com/openshift/hypershift/control-plane-operator/controllers/hostedapicache"
 	"github.com/openshift/hypershift/control-plane-operator/controllers/hostedcontrolplane/manifests"
+	hyperutil "github.com/openshift/hypershift/hypershift-operator/controllers/util"
```

Comment:
@ironcladlou we are wanting to avoid cross-component deps yes?

Reply:
Yeah, neither of the operators should be importing one another. Not automatically enforced since we reverted submodules, so we'll have to remain careful during reviews. Thanks.

Just duplicate the code for now as necessary.